src/pyseekdb/client/client_base.py

```
@@ -1664,18 +1674,18 @@ def _collection_query(
             sql = f"""
-                SELECT {select_clause}, 
-                       {distance_func}(embedding, '{vector_str}') AS distance
+                SELECT {select_clause},
+                       {distance_func}(embedding, {vector_str}) AS distance
                 FROM `{table_name}`
                 {where_clause}
-                ORDER BY {distance_func}(embedding, '{vector_str}')
+                ORDER BY {distance_func}(embedding, {vector_str})
                 APPROXIMATE
                 LIMIT %s
             """
```

Ruff (0.14.8)

Possible SQL injection vector through string-based query construction

(S608)